ush/ioda/bufr2ioda/bufr2ioda_conventional_prepbufr_ps.py

```diff
@@ -658,6 +704,24 @@ def bufr_to_ioda(config, logger):
         .write_attr('long_name', 'Observation Type') \
         .write_data(typ_ps)
 
+    # ObsSubType - airTemperature
+    obsspace.create_var('obsSubType/airTemperature', dtype=obsSubType.dtype,
```

Comment:
Shall we change `obsSubType` to `ObsSubType` since we have `ObsType`?

Reply:
I checked the output and found that the variable type is int64.  We should use int32.  
```
group: ObsSubType {
  variables:
        int64 airTemperature(Location) ;
                airTemperature:_FillValue = 999999LL ;
                string airTemperature:long_name = "Observation SubType" ;
        int64 stationPressure(Location) ;
                stationPressure:_FillValue = 999999LL ;
                string stationPressure:long_name = "Observation SubType" ;
        int64 virtualTemperature(Location) ;
                virtualTemperature:_FillValue = 999999LL ;
                string virtualTemperature:long_name = "Observation SubType" ;
```